agents/trials.js

```diff
@@ -57,8 +57,13 @@ function searchTrials(query, page, perPage, filters) {
     .then((response) => response.obj);
 }
 
+function searchTrialsByEntity(entityType, entityName) {
+  return searchTrials(`${entityType}:${entityName}`, 1, 10);
```

Comment:
That would be ideal, but unfortunately isn't possible. For example, say that the API is called like http://api.opentrials.net/v1/search?q=location:united%20kingdom, we can't know from the API if the original query should've been `location:"united kingdom"` or `location:united kingdom` (i.e. `kingdom location:united`). The only place that knows what the query should be is the Explorer, so this logic needs to be there.